inception-pdf-editor/src/main/java/de/tudarmstadt/ukp/inception/pdfeditor/pdfanno/model/PdfExtractFile.java

```diff
@@ -199,6 +209,12 @@ private void initializeStringContent(String aPdftxt)
             stringToExtract.put(strContentIndex, extractLineIndex);
             extractToString.put(extractLineIndex, strContentIndex);
 
+            if (page > lastPage) {
+                pageOffsetMap.put(lastPage, new Offset(pageBeginIndex, extractLineIndex - 1));
+                lastPage = page;
+                pageBeginIndex = extractLineIndex;
+            }
+
             // if value of PdfExtractLine is in brackets it is a draw operation and is ignored
             if (!extractLine.getValue().matches("^\\[.*\\]$")
                 && !extractLine.getValue().equals("NO_UNICODE"))
```

Comment:
Can you add a comment when `NO_UNICODE` appears?

Reply:
It is in the PdfExtract file.
It is used "when unicode mapping is unavailable".